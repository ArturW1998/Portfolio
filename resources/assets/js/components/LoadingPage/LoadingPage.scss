.loading_page {
  align-items: center;
  display: flex;
  flex-direction: column;
  height: 100vh;
  justify-content: center;

  .loading_text {
    -webkit-animation: loading_text 3s linear infinite;
    animation: loading_text 3s linear infinite;
    color: #87ceeb;
    font-size: 20px;
    text-shadow: 0 0 10px #87ceeb, 0 0 15px #87ceeb;
  }

  @-webkit-keyframes loading_text {
    0% {
      text-shadow: 0 0 0 #87ceeb, 0 0 0 #87ceeb;
    }

    50% {
      text-shadow: 0 0 10px #87ceeb, 0 0 15px #87ceeb;
    }

    100% {
      text-shadow: 0 0 0 #87ceeb, 0 0 0 #87ceeb;
    }
  }

  @keyframes loading_text {
    0% {
      text-shadow: 0 0 0 #87ceeb, 0 0 0 #87ceeb;
    }

    50% {
      text-shadow: 0 0 10px #87ceeb, 0 0 15px #87ceeb;
    }

    100% {
      text-shadow: 0 0 0 #87ceeb, 0 0 0 #87ceeb;
    }
  }
}
