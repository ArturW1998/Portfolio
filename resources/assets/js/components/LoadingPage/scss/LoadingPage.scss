.loading_page {
  align-items: center;
  display: flex;
  flex-direction: column;
  height: 100vh;
  justify-content: center;

  .loading_text {
    color: #87ceeb;
    font-size: 20px;
  }
}
