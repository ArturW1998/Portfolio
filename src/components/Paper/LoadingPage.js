import React from 'react';
import Paper from './Paper';

const Loading = () => <Paper>Loading ...</Paper>;

export default Loading;
