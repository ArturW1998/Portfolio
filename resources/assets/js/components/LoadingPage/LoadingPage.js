import React from 'react';

import './scss/LoadingPage.scss';

const LoadingPage = () => (
  <div className="loading_page for_fade">
    <p className="loading_text">Loading ...</p>
  </div>
);

export default LoadingPage;
