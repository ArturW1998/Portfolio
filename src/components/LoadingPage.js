import React from 'react';
import Paper from './Paper/Paper';

const Loading = () => <Paper>Loading ...</Paper>;

export default Loading;
