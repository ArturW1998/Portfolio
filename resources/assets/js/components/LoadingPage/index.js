import React from 'react';
import './LoadingPage.scss';

const LoadingPage = () => (
  <div className='loading_page for_fade'>

    <p className='loading_text'>Loading ...</p>

  </div>
);

export default LoadingPage;
