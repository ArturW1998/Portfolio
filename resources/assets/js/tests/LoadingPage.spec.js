import React from 'react';

import LoadingPage from '../components/LoadingPage/LoadingPage';

describe('LoadingPage', () => {
  const wrapper = shallow(<LoadingPage />);

  it('Renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
