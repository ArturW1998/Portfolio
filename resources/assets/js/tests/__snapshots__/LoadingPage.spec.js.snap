// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage Renders properly 1`] = `
<div
  className="loading_page for_fade"
>
  <p
    className="loading_text"
  >
    Loading ...
  </p>
</div>
`;
